bin/mono

```diff
@@ -24,7 +35,7 @@ if $?.success?
     exit 1
   end
 
-  current_branch = MonoCheck.run("git rev-parse --abbrev-ref HEAD").chomp
+  current_branch = MonoCheck.run!("git rev-parse --abbrev-ref HEAD").chomp
```

Comment:
Can the command fail that it needs the `run!` variant?

Reply:
I don't think so! But if it does, I'd rather it fail than silently do the wrong thing? I'm using `run!` where I don't expect commands to fail, and `run` with manually-handled `$?.success?` where I do.

Reply:
ok 👍